### ISSUE TITLE
Path to Renpy project not accepted because it contains spaces

### ISSUE THREAD

**Issue by SergioAlonsoJr** [kobaltcore/renconstruct#6]
I tried using renconstruct with both absolute path and relative path:

`renconstruct -c "D:/VN Dropbox/Escrita/build_and_upload/steam/config_steam.yml" -i "d:/VN Dropbox/Escrita/visual-novel-demo" -o "C:/vn_local_data/renpy_builds" -d`

`renconstruct -c "config_steam.yml" -i "../../visual-novel-demo" -o "C:/vn_local_data/renpy_builds" -d`

but in both cases I get an error message like this, exactly where there is a space in my path

```
   DEBUG    renpy.py: error: unrecognized           renconstruct.py:436
                    arguments:                                                 
                    Dropbox\Escrita\visual-novel-demo                 
```         

If I copy and paste my project folder to a path without spaces it works

`renconstruct -c "D:/VN Dropbox/Escrita/build_and_upload/steam/config_steam.yml" -i "C:/vn_local_data/renpy_project" -o "C:/vn_local_data/renpy_builds" -d`

**Comment by kobaltcore** [comment]
This might be an internal issue when it comes to quoting command line arguments correctly, I'll investigate this over the coming days and will report back once it's resolved.

**Comment by kobaltcore** [comment]
This should now be fixed in `v1.9.0` which is currently being pushed out.
